docs/timeseries/analysis.rst

```diff
@@ -0,0 +1,213 @@
+    >>> ts.sort('flux')
+    >>> ts
+    <TimeSeries length=5>
+              time            flux
+             object         float64
+    ----------------------- -------
+    2016-03-22T12:30:31.000     1.0
+    2016-03-22T12:30:43.000     2.0
+    2016-03-22T12:30:40.000     3.0
+    2016-03-22T12:30:34.000     4.0
+    2016-03-22T12:30:37.000     5.0
+
+Resampling
+==========
+
+We provide a :func:`~astropy.timeseries.simple_downsample` function
+that can be used to bin values from a time series into bins of equal time, using
+a custom function (mean, median, etc.). This operation returns a
+|BinnedTimeSeries|. Note that this is a simple function in the sense that it
+does not for example know how to treat columns with uncertainties differently
+from other values, and it will blindly apply the custom function specified to
+all columns.
+
+The following example shows how to use this to bin a light curve from the Kepler
+mission into 20 minute bins using a median function. First, we read in the data
+using:
+
+.. plot::
+   :context: reset
+   :nofigs:
+
+    from astropy.timeseries import TimeSeries
+    from astropy.utils.data import get_pkg_data_filename
+    example_data = get_pkg_data_filename('timeseries/kplr010666592-2009131110544_slc.fits')
+    kepler = TimeSeries.read(example_data, format='kepler.fits')
+
+(see :ref:`timeseries-io` for more details about reading in data). We can then
+downsample using:
+
+.. plot::
+   :include-source:
+   :context:
+   :nofigs:
+
+    import numpy as np
+    from astropy import units as u
+    from astropy.timeseries import simple_downsample
+    kepler_binned = simple_downsample(kepler, time_bin_size=20 * u.min, func=np.nanmedian)
+
+We can take a look at the results:
+
+.. plot::
+   :include-source:
+   :context:
+
+    import matplotlib.pyplot as plt
+    plt.plot(kepler.time.jd, kepler['sap_flux'], 'k.', markersize=1)
+    plt.plot(kepler_binned.time_bin_start.jd, kepler_binned['sap_flux'], 'r-', drawstyle='steps-pre')
+    plt.xlabel('Barycentric Julian Date')
+    plt.ylabel('SAP Flux (e-/s)')
+
+Folding
+=======
+
+The |TimeSeries| class has a
+:meth:`~astropy.timeseries.TimeSeries.fold` method that can be used to
+return a new time series with a relative and folded time axis. This method
+takes the period as a :class:`~astropy.units.Quantity`, and optionally takes
+an epoch as a :class:`~astropy.time.Time`, which defines a zero time offset:
+
+.. plot::
+   :context: reset
+   :nofigs:
+
+   import numpy as np
+   from astropy import units as u
+   import matplotlib.pyplot as plt
+   from astropy.timeseries import TimeSeries
+   from astropy.utils.data import get_pkg_data_filename
+
+   example_data = get_pkg_data_filename('timeseries/kplr010666592-2009131110544_slc.fits')
+   kepler = TimeSeries.read(example_data, format='kepler.fits')
+
+.. plot::
+   :include-source:
+   :context:
+
+    kepler_folded = kepler.fold(period=2.2 * u.day, midpoint_epoch='2009-05-02T20:53:40')
```

Comment:
As someone with no experience in exoplanet research, the values you picked here seem like magic numbers to me. Is explaining why you pick those values out of topic?

Reply:
I added a sentence about this and linked to a page showing how to do the periodogram.